pkg/provider/branch/trunk/trunk_test.go

```diff
@@ -847,3 +847,20 @@ func TestTrunkENI_CreateAndAssociateBranchENIs_ErrorCreate(t *testing.T) {
 	assert.Error(t, MockError, err)
 	assert.Equal(t, []*ENIDetails{EniDetails1}, trunkENI.deleteQueue)
 }
+
+func TestTrunkENI_Introspect(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	trunkENI, _, mockInstance := getMockHelperInstanceAndTrunkObject(ctrl)
+	trunkENI.trunkENIId = trunkId
+	trunkENI.uidToBranchENIMap[PodUID] = branchENIs1
```

Comment:
A variable naming question. Does `s1` here denote anything in the global var list?

Reply:
It's bad naming convention from my side. It means the first branch ENI ID. I will fix this as well in follow up PR.